doc/whats_new/v0.24.rst

```diff
@@ -235,6 +235,10 @@ Changelog
   will be removed in 0.28. :pr:`17662` by
   :user:`Joshua Newton <joshuacwnewton>`.
 
+- |Fix| Fix :issue:`10493`. Fix Local Linear Embedding (LLE) 
```

Comment:
I wonder if this should be Efficiency since there is no faulty logic as such, rather we have reduced the space complexity.

Reply:
This can make sense, i don't know all the categories :)
I pushed a new version